zephyr/wrapper.c

```diff
@@ -746,7 +746,12 @@ int arch_cpu_secondary_cores_prepare_d0ix(void)
 
 void arch_cpu_disable_core(int id)
 {
-	/* TODO: call Zephyr API */
```

Comment:
Dont we need to keep the TODO since the end goal is the Zephyr API when it's ready ?

Reply:
@lgirdwood  I dropped as we have no TODOs in the enable_core either. The whole wrapper.c needs to finally disappear, so I don't think the TODOs add much value here.